src/mergeWith.js

```diff
@@ -0,0 +1,33 @@
+var _curry3 = require('./internal/_curry3');
+var mergeWithKey = require('./mergeWithKey');
+
+
+/**
+ * Creates a new object with the own properties of the two provided objects. If
+ * a key exists in both objects, the provided function is applied to the values
+ * associated with the key in each object, with the result being used as the
+ * value associated with the key in the returned object. The key will be
+ * excluded from the returned object if the resulting value is `undefined`.
+ *
+ * @func
+ * @memberOf R
+ * @since x.y.z
```

Comment:
No. I see two options:

  - specify the next minor version; or
  - leave it out and rely on a prepublish script to add it.

The first option is easy to understand, but will potentially require a much-debated pull request to be updated several times as releases are made while it is waiting to be merged.

The second option is arguably more reliable, but I haven't written the script yet. ;)